extension/encoding/awslogsencodingextension/README.md

```diff
@@ -307,6 +314,24 @@ If you're using the old format values you should update the encoding extension c
 | `userIdentity.sessionContext.sessionIssuer.accountId`     | `aws.user_identity.session_context.issuer.account_id`                     |
 | `userIdentity.sessionContext.sessionIssuer.userName`      | `aws.user_identity.session_context.issuer.user_name`                      |
 
+#### CloudTrail digest record
+
+[CloudTrail digest record](https://docs.aws.amazon.com/awscloudtrail/latest/userguide/cloudtrail-log-file-validation-digest-file-structure.html) get mapped with following attributes in the resulting OpenTelemetry log:
+
+| CloudTrail digest field  | Attribute in OpenTelemetry log                    |
```

Comment:
Can you please list the fields from https://docs.aws.amazon.com/awscloudtrail/latest/userguide/cloudtrail-log-file-validation-digest-file-structure.html that we're not translating here, and mention they're not stored?

I don't follow the rationale from the description about why we don't store the signature/hash/algorithm. I'd rather capture it and let people opt out of storing it (e.g. remove with transform processor), than not capture it at all.